.github/workflows/doc-pr-cherry-pick.yml

```diff
@@ -0,0 +1,69 @@
+name: Cherry-Pick PR to v2
+
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'PR number to cherry-pick'
+        type: string
+        required: true
+
+jobs:
+  cherry_pick_and_create_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
Use SHAs to pin versions, like this:

https://github.com/aws/aws-cli/blob/4f42c0921fde873f16a947bc306dc88528ae19b3/.github/workflows/run-dep-tests.yml#L20

Reply:
Addressed via https://github.com/aws/aws-cli/pull/9142/commits/96705c89ada596eebcacf55b8a706793f3488ec4